### ISSUE TITLE
Snippet editor: improve aria-labelledby and aria-describedby usage

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#601]
Noticed while CR'ing #600 and #545 

## aria-labelledby

In https://github.com/Yoast/yoast-components/pull/545/files#r196005254 the `aria-labelledby` value of the Title and Description fields were simplified and they're now just a number. While, if I remember correctly, ID values that start with a number are perfectly valid in HTML5 (they were not in HTML4), I'd recommend to consider to add a prefix or something to better qualify these values and avoid potential conflicts with other IDs.

Also, there's a bit of inconsistency with the slug field which still uses a prefix + number.

![screen shot 2018-06-18 at 10 21 37](https://user-images.githubusercontent.com/1682452/41527414-5d88ea5e-72e7-11e8-8276-ccad5f5d44c9.png)

![screen shot 2018-06-18 at 11 01 49](https://user-images.githubusercontent.com/1682452/41527415-5da519d6-72e7-11e8-8a16-12e076268511.png)

== aria-describedby

This aria property is used to associate a field to the description provided via the "visual placeholder" we're using on the description field:

<img width="750" alt="screen shot 2018-06-18 at 10 46 30" src="https://user-images.githubusercontent.com/1682452/41527466-8d7969b4-72e7-11e8-97f4-e0a299f1799d.png">

That's good, however also the Title field has an `aria-describedby` attribute which points to nothing, as there's no description for the title field. `aria-describedby` should always reference other existing elements.

**Comment by afercia** [comment]
For the `aria-describedby` still rendered even when there's no placeholder, see https://github.com/facebook/draft-js/issues/1739

**Comment by afercia** [comment]
Worth noting there's also a warning
`Warning: Failed prop type: Invalid prop `ariaLabelledBy` of type `number` supplied to `ReplacementVariableEditorStandalone`, expected `string`.`
